pandunia-lekse.csv

```diff
@@ -2133,6 +2161,7 @@ prasedim|mate|Pr||praseodymium||praseodimio||||||||||||||praseodyymi|
 prate|||eng:practice, fra:pratique, por:prática, tur:pratik, pol:praktyka, hin:प्रथा (prathā)|practice (actuality)||práctica||||||||||||||käytäntö (pragmatia)|praktyka, aktualność
 prati||||practical (pragmatic)||práctico||||||||||||||käytännöllinen (pragmaattinen)|practyczny, pragmatyczny
 preda|coverb||rus:п(е)ред (p(e)red), pol:przed, eng:spa:por:pre-, fra:pré-|precede (come before)||preceder (anteceder)|||||||||||||antaŭiri|edeltää (tulla ennen)|poprzedzić, poprzedzać
+prede||||lead-up|||||||||||||||||
```

Comment:
Beforeness? Front?

Reply:
Actually, perhaps "prede" could mean front instead of "jine".

Reply:
si :)